src/frontend/src/lib/components/pages/SettingsSignedIn.svelte

```diff
@@ -15,23 +15,35 @@
 	import { fade } from 'svelte/transition';
 	import { busy } from '$lib/stores/busy.store';
 	import { POUH_ENABLED } from '$lib/constants/credentials.constants';
+	import type { OptionIdentity } from '$lib/types/identity';
+	import { loadUserProfile } from '$lib/services/load-user-profile.services';
+	import { userProfileStore } from '$lib/stores/settings.store';
 
 	let remainingTimeMilliseconds: number | undefined;
 	$: remainingTimeMilliseconds = $authRemainingTimeStore;
 
+	let identity: OptionIdentity;
+	$: identity = $authStore?.identity;
+
 	let principal: Principal | undefined | null;
-	$: principal = $authStore?.identity?.getPrincipal();
+	$: principal = identity?.getPrincipal();
 
 	const getPouhCredential = async () => {
-		if (nonNullish(principal)) {
+		if (nonNullish(identity)) {
 			try {
 				busy.show();
-				await requestPouhCredential({ credentialSubject: principal });
+				await requestPouhCredential({ identity });
 			} finally {
 				busy.stop();
 			}
 		}
 	};
+
+	$: {
+		if (nonNullish(identity) && POUH_ENABLED) {
```

Comment:
`authSignedIn` instead of `nonNullish(identity)` probably

Reply:
I'd still need `nonNullish(identity)` because TS doesn't know that when `authSignedIn` is true, `identity` is defined.

Reply:
You can pass a `OptionIdentity` all along and the initialization of the actor ultimately throws an error.